addon/mixins/regenerated/models/new-platform-flexberry-g-i-s-map.js

```diff
@@ -6,24 +6,50 @@ import Ember from 'ember';
 import DS from 'ember-data';
 import { Projection } from 'ember-flexberry-data';
 
-/**
-  Mixin containing map model attributes, relations & projections.
-
-  @class NewPlatformFlexberyGISMapModelMixin
-  @extends <a href="http://emberjs.com/api/classes/Ember.Mixin.html">Ember.Mixin</a>
-*/
```

Comment:
??

Reply:
Result of regeneration. I'll try to restore them.
But each regenaretion will remove comments again.